projects/schema-form/src/lib/model/nullproperty.ts

```diff
@@ -0,0 +1,25 @@
+import { AtomicProperty } from './atomicproperty';
+import { ValidatorRegistry, PropertyGroup } from '.';
+import { ExpressionCompilerFactory } from '../expression-compiler-factory';
+import { SchemaValidatorFactory } from '../schemavalidatorfactory';
+import { PROPERTY_TYPE_MAPPING } from './typemapping';
+import {ISchema} from './ISchema';
+import { LogService } from '../log.service';
+
+export class NullProperty extends AtomicProperty {
+  fallbackValue() {
+    return null;
+  }
+}
+
+PROPERTY_TYPE_MAPPING.boolean = (
```

Comment:
`boolean` name is already in use by `BooleanProperty` model. you meant `PROPERTY_TYPE_MAPPING.null = ...`, perhaps

Reply:
yep, it's mistake. will change on null key

Reply:
:white_check_mark:  fixed